src/molecule.py

```diff
@@ -32,10 +32,6 @@
 except NameError:
     pass
 
-# Special error which is thrown when TINKER .arc data is detected in a .xyz file
```

Comment:
This needs to be reverted, perhaps by merging the master branch into this branch.

Reply:
(As in, the "red" part needs to be put back and the "green" part needs to be removed.)